src/audit/index.js

```diff
@@ -0,0 +1,79 @@
+import axios from 'axios';
+import { errorHandler } from '../utils';
+
+const pdp = (base, token) => {
+  base = `${base}`;
+  var instance = axios.create({
```

Comment:
@nopbyte could you rebase this with #30? And then remove the use of instance?

Reply:
Actually you still may want to use an instance in this case because you are returning `res.data.result` instead of the global default of `res.data`. 

You can override the global response instance by doing this: 
```
var instance = axios.create();
instance.interceptors.response.use((res) => {
     return res.data.result
});
```

And then you're function would look like this: 
```
getUserActions: (params) => {
      let url = `${base}/api/v1/audit/actions/byMe`;
      return instance.request({
        method: 'GET',
        url: url
      })
},
```